src/runtime-rs/crates/hypervisor/src/device/vsock.rs

```diff
@@ -15,3 +20,55 @@ pub struct VsockConfig {
     /// unix domain socket path
     pub uds_path: String,
 }
+
+#[derive(Debug)]
+pub struct VsockConfig {
+    /// Unique identifier of the device
+    pub id: String,
+
+    /// A 32-bit Context Identifier (CID) used to identify the guest.
+    pub guest_cid: u32,
+
+    pub vhost_fd: File,
+}
+
+const VHOST_VIRTIO: u8 = 0xAF;
+nix::ioctl_write_ptr!(vhost_vsock_set_guest_cid, VHOST_VIRTIO, 0x60, u64);
```

Comment:
Why is this not in a function call? Also, please add a comment explaining what it's doing ;)

Reply:
The comment is added

Reply:
Nit: For consistency with the `const` above and for more clarity, I'd consider adding something like:

```rust
// From /usr/include/linux/vhost.h
const VHOST_VSOCK_SET_GUEST_CID: u8 = 0x60;
```